vscode/test/suite/testUtils.ts

```diff
@@ -43,3 +44,34 @@ export const app1 = {
 
 export const strDevice1 = `{"configured_firmware":"","device_id":"12345678-1234-1234-1234-12345678","is_active":false,"is_simulator":false,"last_seen":"1989-06-03T10:00:00Z","name":"test1","running_firmware":"v1.1.2"}`;
 export const strDevice2 = `{"configured_firmware":"","device_id":"01234567-0123-0123-0123-01234567","is_active":false,"is_simulator":false,"last_seen":"1989-07-03T10:00:00Z","name":"test2","running_firmware":"v1.1.2"}`;
+
+export const pkg1 = {
+  "name":"test1",
+  "description":"This is a package.",
+  "license":"MIT",
+  "url":"github.com/toitware/test",
+  "version":"1.2.0",
+  "dependencies": []
+} as ConsolePackage;
+
+export const pkg2 = {
+  "name":"test-2",
+  "description":"This is a package.",
+  "license":"MIT",
+  "url":"github.com/toitware/test-2",
+  "version":"1.1.2",
+  "dependencies":[{"url":"github.com/toitware/test", "version":"^1.1.0"} as ConsoleDependency]
```

Comment:
Normally `as` feels like something that should be avoided. Is there a better way to type the object?
For example, I wouldn't be sure that the compiler complains if a field changes name...

Reply:
I think the compiler (or maybe the linter) complains if you use as and the objects do not look the same.

Reply:
Maybe it would work with this syntax?
https://stackoverflow.com/a/19457036/997251

Reply:
Hmm. It seems like that's as bad as `as`.
Maybe assigning it to a temporary is the only way... :(